.github/workflows/workflow.yml

```diff
@@ -0,0 +1,32 @@
+# name: coveralls
```

Comment:
Coveralls was set up for the metaAssistant repository, now we are in the home repository it could be set up again and I thought there was no urgency for this so it could be a good trainee task. This workflow will be needed for this task but it's commented out to prevent the github check failing until it is set up.

Reply:
I have removed for now and put as a reference in the trello task